src/client/features/search/query-entity-info.js

```diff
@@ -188,66 +216,60 @@ const queryEntityInfo = query => {
       uniprotIds[ entityId ] = entityId;
     } else if( isNcbiId( token ) ){
       ncbiIds[ entityId ] = entityId;
-      genes.push( entityId );
+    } else if (isHgncId( token ) ){
+      hgncId[ entityId ] = entityId;
     } else {
+      prefix = false;
```

Comment:
what is the purpose of this prefix variable?

Reply:
To determine whether the query has <database>:<id> format. If not, validate the id in gProfiler.

Reply:
So queries like:

`ncbi:1 uniprot:p06777 hgnc:tp53` will get the provider information for each of these tokens but

`ncbi:1 uniprot:p06777 hgnc:tp53 blah` will send all of these ids to the validator?  

I think it makes sense to create two lists of tokens.  One list contains ['uniprot:p06777', ... 'ncbi:1'], and the other contains ['blah'].  Get the provider specific info for each token in the first list, and validate each token from gProfiler in the second list.  @jvwong is that what you meant?

Reply:
Do we consider the case `ncbi:1 uniprot:p06777 hgnc:tp53 blah`? I assume the system only accepts either:
1. `<db>:<id>` (only one token)
2. `<ids>` (one or more tokens)

Reply:
The prefix is only valid when a single token is available. Else send to validator.

Reply:
If that is the case, I think the code should reflect that instead of checking each token to see if it is special.